channel/state/outcome/outcome_test.go

```diff
@@ -58,3 +153,25 @@ func TestExitEncode(t *testing.T) {
 	}
 
 }
+
+func TestExitDecode(t *testing.T) {
+	var zeroBytes = []byte{}
+	var encodedExit, _ = hex.DecodeString("00000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000060000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000002000000000000000000000000096f7123e3a80c9813ef50213aded0e4511cb820f0000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000000")
+
+	var decodedExit, err = Decode(encodedExit)
+	if err != nil {
+		t.Error(err)
+	}
+
+	var a = Allocations{{
+		Destination:    common.HexToHash("0x00000000000000000000000096f7123E3A80C9813eF50213ADEd0e4511CB820f"),
+		Amount:         big.NewInt(1),
+		AllocationType: 0,
+		Metadata:       zeroBytes}}
```

Comment:
Could we / should we dry out this test by putting reused variables at the top of the file (outer scope)?